lib/danger/request_source/bitbucket_server_api.rb

```diff
@@ -0,0 +1,70 @@
+# coding: utf-8
+require "danger/helpers/comments_helper"
+
+module Danger
+  module RequestSources
+    class BitbucketServerAPI
```

Comment:
Strictly speaking this isn't a request source so having in in `lib/danger/request_source` might be confusing. Can we put it somewhere else?